orangecontrib/imageanalytics/image_embedder.py

```diff
@@ -193,11 +196,21 @@ def _send_to_server(self, file_paths, image_processed_callback):
         )
 
     def _load_image_or_none(self, file_path):
-        try:
-            image = open_image(file_path)
-        except IOError:
-            log.warning("Image skipped (invalid file path)", exc_info=True)
-            return None
+	o = urlparse(file_path)
```

Comment:
`o` is a bit ambiguous. What would be a more descriptive name for this variable?

# Reasoning
The book [Clean Code: A Handbook of Agile Software Craftsmanship](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882) has many good practices. The following are relevant here (with examples in the JavaScript language):

- [Use explanatory variables](https://github.com/ryanmcdermott/clean-code-javascript#use-explanatory-variables)
- [Avoid Mental Mapping](https://github.com/ryanmcdermott/clean-code-javascript#avoid-mental-mapping) - explicit is better than implicit (also from Zen of Python)